lib/resources/example_json/r4_examples_medication_request.rb

```diff
@@ -164,6 +341,3830 @@ module Resources
+                {
+                  'valueCode': 'unknown',
+                  'url': 'http://hl7.org/fhir/StructureDefinition/data-absent-reason'
+                }
+              ]
+            },
+            'dosageInstruction': [
+              {
+                'text': '1 patches, Topical, every week',
+                'patientInstruction': '1 patch(es) Topical (on the skin) every week. Refills: 0.',
+                'timing': {
+                  'repeat': {
+                    'boundsPeriod': {
+                      'start': '2020-08-12T09:00:00.000-05:00'
+                    }
+                  },
+                  'code': {
+                    'coding': [
+                      {
+                        'system': 'https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/codeSet/4003',
+                        'code': '696617',
+                        'display': 'every week'
+                      },
+                      {
+                        'system': 'http://terminology.hl7.org/CodeSystem/v3-GTSAbbreviation',
+                        'code': 'WK',
+                        'display': 'weekly'
+                      },
+                      {
+                        'system': 'http://snomed.info/sct',
+                        'code': '225769003',
+                        'display': 'Once a week (qualifier value)'
+                      }
+                    ],
+                    'text': 'every week'
+                  }
+                },
+                'route': {
+                  'coding': [
+                    {
+                      'system': 'https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/codeSet/4001',
+                      'code': '318190',
+                      'display': 'Topical',
+                      'userSelected': true
+                    },
+                    {
+                      'system': 'http://snomed.info/sct',
+                      'code': '6064005',
+                      'display': 'Topical route (qualifier value)',
+                      'userSelected': false
+                    }
+                  ],
+                  'text': 'Topical'
+                },
+                'doseAndRate': [
+                  {
+                    'doseQuantity': {
+                      'value': 1,
+                      'unit': 'patches',
+                      'system': 'http://unitsofmeasure.org',
+                      'code': '{Patch}'
+                    }
+                  }
+                ]
+              }
+            ],
+            'dispenseRequest': {
+              'validityPeriod': {
+                'start': '2020-08-12T08:48:57.000-05:00'
+              }
+            },
+            'priorPrescription': {
+              'reference': 'MedicationRequest/314010037'
+            }
+          }
+        },
+        {
+          'fullUrl': 'https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/MedicationRequest/311877117',
```

Comment:
These should be closed endpoints
https://fhir-ehr.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/MedicationRequest

Reply:
comment addressed